application/django/README.md

```diff
@@ -0,0 +1,24 @@
+Init the SecretHub demo repo with example values
+```
+secrethub demo init
+```
+
+Create a service account for the demo repo
+```
+secrethub service init --description demo_service \
+--permission read --file demo_service.cred <your_username>/demo
+```
+
+Build the sinatra docker demo
+```
+docker build . -t django:demo
+```
+
+Run the docker demo with the secrets in the environment variables
+```
+docker run -ti -p 8080:8000 \
+  -e DEMO_USERNAME=secrethub://<your_username>/demo/username \
+  -e DEMO_PASSWORD=secrethub://<your_username>/demo/password \
+  -e SECRETHUB_CREDENTIAL=$(cat demo_service.cred) \
+  django:demo
+```
```

Comment:
How can I check that it is actually working correctly?

Reply:
Added in the README.md

Reply:
I expected this to be added here as the last step, something like:

```
If you now visit http://localhost:8080, you should see the welcome message including your username.
```

Reply:
Added